[2023-10-01 05:57:36,833] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_third_dag_v3.second_task 2021-08-29T00:00:00+00:00 [queued]>
[2023-10-01 05:57:37,159] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_third_dag_v3.second_task 2021-08-29T00:00:00+00:00 [queued]>
[2023-10-01 05:57:37,184] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 05:57:37,186] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-10-01 05:57:37,188] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 05:57:37,545] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-08-29T00:00:00+00:00
[2023-10-01 05:57:37,841] {standard_task_runner.py:52} INFO - Started process 2733 to run task
[2023-10-01 05:57:38,105] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_third_dag_v3', 'second_task', '2021-08-29T00:00:00+00:00', '--job-id', '247', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_third_dag.py', '--cfg-path', '/tmp/tmpahm5obls', '--error-file', '/tmp/tmpcrt59nfk']
[2023-10-01 05:57:38,484] {standard_task_runner.py:77} INFO - Job 247: Subtask second_task
[2023-10-01 05:57:39,480] {logging_mixin.py:104} INFO - Running <TaskInstance: our_third_dag_v3.second_task 2021-08-29T00:00:00+00:00 [running]> on host d4ed655215d1
[2023-10-01 05:57:40,584] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_third_dag_v3
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-29T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-29T00:00:00+00:00
[2023-10-01 05:57:40,620] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-10-01 05:57:40,647] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2023-10-01 05:57:40,854] {bash.py:169} INFO - Output:
[2023-10-01 05:57:40,878] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2023-10-01 05:57:40,891] {bash.py:177} INFO - Command exited with return code 0
[2023-10-01 05:57:41,318] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_third_dag_v3, task_id=second_task, execution_date=20210829T000000, start_date=20231001T055736, end_date=20231001T055741
[2023-10-01 05:57:43,129] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-01 05:57:43,805] {local_task_job.py:146} INFO - Task exited with return code 0
